modules/configure-ocp-oauth-identity-provider.adoc

```diff
@@ -22,5 +23,5 @@ To integrate the built-in {ocp} OAuth server as an identity provider for {produc
 ====
 For security, Red Hat recommends setting the *Minimum access role* to *None* while you complete setup. Later, you can return to the *Access Control* page to set up more tailored access rules based on user metadata from your identity provider.
 ====
-. To add access rules for users and groups accessing {product-title-short}, use the *Rules* section. For example, to give the *Admin* role to a user called `administrator`, select *User* as *Key*, *administrator* as *Value*, and *Admin* as the *Role*. Use *Add new rule* to add more rules.
+. To add access rules for users and groups accessing {product-title-short}, use the *Rules* section. For example, to give the *Admin* role to a user called `administrator`, select *Name* as *Key*, *administrator* as *Value*, and *Admin* as the *Role*. Use *Add new rule* to add more rules.
```

Comment:
Please mention that `email` attribute is not set for this auth provider because of https://docs.openshift.com/container-platform/4.9/rest_api/user_and_group_apis/user-user-openshift-io-v1.html

Reply:
Added!